Altari.Infrastructure.Logger/Helper/EmailLogger.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System;
+using System.Configuration;
+using System.Net;
+using System.Net.Mail;
+
+namespace Onion.Infrastructure.ApplicationLog.Helper
+{
+    public static class EmailLogger
+    {
+        public static bool SendEmail(string body)
+        {
+            var to = new MailAddress(ConfigurationManager.LOG4NET_EMAIL_TO);
+
+            var from = new MailAddress(ConfigurationManager.LOG4NET_EMAIL_FROM);
+
+            var mail = new MailMessage(from, to);
+
+            mail.Subject = ConfigurationManager.LOG4NET_EMAIL_SUBJECT;
+
+            mail.Body = ConfigurationManager.LOG4NET_EMAIL_BODY;
+
+            SmtpClient smtp = new SmtpClient();
```

Comment:
Should we throw this in a using statement so it's disposed of appropriately? It feels like it might be a little heavy?

Reply:
I think we can create another library called "Onion.Infrastructure.Email" and reuse this for sending emails